kubernetes/deployment-staging.tmpl

```diff
@@ -6,6 +6,11 @@ metadata:
     app: caesar-staging-app
 spec:
   replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
```

Comment:
What's the reason for limiting this to 1? I guess it won't make any difference to staging, but in production the default of 25% would mean quicker deploys.

Reply:
I want 1 container with the new code to launch so i've hardcoded it to 1. Based my reading, this will mean we've got 1 pod with old code serving traffic and 1 pod with new code which will boot and migrate and then start serving traffic.